triage_bot/lib/engine.dart

```diff
@@ -60,11 +60,14 @@ sealed class GitHubSettings {
     'go_router',
     'infra',
     'ios',
+    'linux',
+    'macos',
```

Comment:
Shouldn't `desktop` (line 55) be removed as part of this?

Reply:
I was assuming I'd remove desktop as a followup once the issue re-triage has concluded.

Reply:
Done.